pkg/didcomm/crypto/jwe/authcrypt/decrypt.go

```diff
@@ -134,11 +120,6 @@ func (c *Crypter) findRecipient(jweRecipients []Recipient, recipientPubKey *[cha
 }
 
 func (c *Crypter) decryptSharedKey(recipientKp jwecrypto.KeyPair, senderPubKey *[chacha.KeySize]byte, recipient *Recipient) ([]byte, error) { //nolint:lll
-	apu, err := base64.RawURLEncoding.DecodeString(recipient.Header.APU)
```

Comment:
This function doesn’t look like it was actually changed in logic.

Reply:
sure, I can pull up apu back here to match this code..

Reply:
I pulled apu DecodeString to its original place